bin/wasabi.sh

```diff
@@ -113,15 +112,17 @@ bootstrap() {
   echo "${green}installed dependencies: ${formulas[@]} ${casks[@]}${reset}"
 }
 
-start() {
-  [ ${build} = "true" ] && ./bin/build.sh -b ${build} -t ${verify}
+build() {
+  ./bin/build.sh -b ${1:-false} -t ${2:-false} -p ${3:-development}
+}
 
+start() {
   ./bin/container.sh -v ${verify} start${1:+:$1}
 }
 
 test() {
```

Comment:
test overrides internal bash function - not very severe since we run in /bin/env bash anyway, but we might consider naming or other solutions.